exercises/practice/dot-dsl/.meta/config.json

```diff
@@ -16,6 +16,9 @@
     "editor": [
       "src/main/java/Node.java",
       "src/main/java/Edge.java"
+    ],
+    "invalidator": [
+      "build.gradle"
```

Comment:
Did you mean to commit this change? It isn't clear to me why we would want to add this to the [invalidator](https://exercism.org/docs/building/tracks/practice-exercises#h-file-meta-config-json).

Reply:
This was part of the `metadata` sync performed by `configlet` with the Problem Specifications repository.

Reply:
Ah, that makes sense! Thanks for letting me know.